app/controllers/api/v1/pizzas_controller.rb

```diff
@@ -1,6 +1,27 @@
 class Api::V1::PizzasController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
   def show
     selected_pizza = Pizza.find(params[:id])
-    render json: { pizza: selected_pizza, brand: selected_pizza.brand }
+    render json: {pizza: selected_pizza, brand: selected_pizza.brand, reviews: selected_pizza.reviews} 
   end
+
+  def new
+    pizza = Pizza.find(params[:id])
+    review = Review.new
+  end
```

Comment:
.......this is pattern matching. We dont need a new endpoint here, this is an api controller! the code isnt being used!